src/main/resources/wa-task-configuration-publiclaw-care_supervision_epo.dmn

```diff
@@ -1121,6 +1122,23 @@
           <text>true</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_1sumgf3">
+        <inputEntry id="UnaryTests_1oc8kos">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0qd59ye">
+          <text>"viewAdditionalApplications","reviewMessageCTSC","reviewMessageLegalAdviser","reviewMessageHearingCentreAdmin","reviewMessageHearingJudge","reviewMessageAllocatedJudge","approveOrders","reviewOrderCMO","reviewOrderCMOUrgent","reviewResponseAllocatedJudge","reviewResponseHearingJudge","reviewResponseHearingCentreAdmin","reviewResponseCTSC"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_19vr5yu">
+          <text>"title"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_040jt2c">
+          <text>if (taskAttributes != null and taskAttributes.name != null) then taskAttributes.name + " (" + substring(string(date(now())), 1, 10) + " " + substring(string(time(now())), 1, 5) + ")" else ""</text>
```

Comment:
**Not a blocker** but else condition worries me slightly, could tasks end up with an empty string as their name, would get confusing, one to check in testing if there's ever a scenario like that

Reply:
if it's not empty string, should I put taskAttributes.name instead? But taskAttributes.name should be null?